programs/mango-v4/src/accounts_ix/openbook_v2_register_market.rs

```diff
@@ -19,11 +19,7 @@ pub struct OpenbookV2RegisterMarket<'info> {
     /// CHECK: Can register a market for any openbook_v2 program
     pub openbook_v2_program: Program<'info, OpenbookV2>,
 
-    /// CHECK: Can register any openbook_v2 market
-    #[account(
-        constraint = openbook_v2_market_external.load()?.base_mint == base_bank.load()?.mint,
-        constraint = openbook_v2_market_external.load()?.quote_mint == quote_bank.load()?.mint,
-    )]
```

Comment:
for register_market we do need to check the mints like this, can you bring it back? (sorry!)